ipn/ipnlocal/cert.go

```diff
@@ -252,7 +254,7 @@ func (b *LocalBackend) getCertStore() (certStore, error) {
 	if err != nil {
 		return nil, err
 	}
-	return certFileStore{dir: dir}, nil
+	return certFileStore{dir: dir, testRoots: testX509Roots}, nil
```

Comment:
Is it worthwhile to panic if we see this non-nil not in a test?

Reply:
Do we do that elsewhere? I don't recall us being that strict with package-level test hooks.

Reply:
But done.